README.md

```diff
@@ -112,13 +112,13 @@ repo init -u https://github.com/Pelagicore/pelux-manifests.git -m <manifest> -b
 repo sync
 ```
 
-When done fetching the sources, create a build directory and set up bitbake.
-TEMPLATECONF tells the `oe-init-build-env` script which path to fetch
-configuration samples from. Note that the example below get the template
-configuration for the Intel BSP, adapt the path according to your current BSP.
+When 'repo sync' has finished fetching the sources, the next step is to create a 'build' directory and set up bitbake.
```

Comment:
I would prefer ` (back ticks) over ' which puts the content in a box making it clearer that you are talking about a command.

Reply:
Done